ssg/build_renumber.py

```diff
@@ -177,7 +177,7 @@ def _add_cce_id_refs_to_oval_checks(self, idmappingdict):
                 "OVAL rule '{0}' doesn't have a description, which is mandatory".format(ovalid)
 
             xccdfcceid = idmappingdict[ovalid]
-            if is_cce_valid(xccdfcceid):
+            if is_cce_format_valid(xccdfcceid) and is_cce_value_valid(xccdfcceid):
```

Comment:
Did we want to provide a `is_cce_valid` wrapper for this along with the test cases?

Else, `tests/unit/ssg-module/test_checks.py` needs to be updated too.

Reply:
Updated the tests.